### PR TITLE
Cosmology Presentation

### PR DESCRIPTION
**Pull request by nstarman**
Help / suggestions appreciated.

TODO:

- [x] tutorial
- [x] exercise~s~

Signed-off-by: Nathaniel Starkman (@nstarman) <nstarkman@protonmail.com>

### CODE REVIEW

**Comment by stargaser** [comment]
Thanks @nstarman! I will take a look and it would be great if @bmorris3 could do that too.

**Review by stargaser — COMMENTED** [review]
Thanks very much for this contribution. I learned a great deal from reading through the notebook. The notebook and the slides look good to me. 

The notebook clearly lays out all the functionality. I did not know about the default cosmology behind distances in coordinates. I also learned more fundamental Python in printing attributes and methods.  This will be a very useful addition.

Given the time pressures for the workshop, I think one exercise will be plenty. I put in some minor comments inline including some more hints for that one. I think adding a solution for the exercise would be all that is needed.

File: 11-Cosmology/Astropy_Cosmology.ipynb [hunk 1]
```diff
@@ -0,0 +1,1038 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercises\n",
+    "\n",
+    "## Exercise 1\n",
+    "\n",
+    "Compute the redshift of matter-radiation equality for the Planck 2018 flat-$\\Lambda$CDM realization."
```
[the remark this answers is not in the document]
Comment (nstarman) [reply 11-Cosmology/Astropy_Cosmology.ipynb]: Good point. I've added hints and uploaded the solutions document.

File: 11-Cosmology/Astropy_Cosmology.ipynb [hunk 2]
```diff
@@ -0,0 +1,1038 @@
...
+    }
+   ],
+   "source": [
+    "fig = plt.figure()\n",
+    "ax = fig.add_subplot(title=\"Perturbed Matter Component\", xlabel=\"Distance [Mpc]\", ylabel=\"z\")\n",
+    "ax.plot(distances, zs.T, c=\"gray\", alpha=0.1);\n",
+    "for z, dm in zip(zs[::10, :], perturb_Om0[::10]):\n",
+    "    ax.plot(distances, z, label=r\"$\\Delta\\Omega_{m0}=$\" + f\"{Planck18.Om0+dm:.3}\")\n",
+    "ax.legend()\n",
+    "fig.show();"
```
[the remark this answers is not in the document]
Comment (nstarman) [reply 11-Cosmology/Astropy_Cosmology.ipynb]: That was a strange one. Changing ``fig.show()`` to ``plt.show()`` did the trick. Odd.

File: 11-Cosmology/Astropy_Cosmology.ipynb [hunk 3]
```diff
@@ -0,0 +1,1038 @@
...
+   "metadata": {},
+   "source": [
+    "# Putting it all together:  a concise example\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's say we have a data set with an array of luminosity distances and we are interested in finding the "
```
[the remark this answers is not in the document]
Comment (nstarman) [reply 11-Cosmology/Astropy_Cosmology.ipynb]: Haha. Forgot to push the that diff. I've added the full text now. It's a bit of a contrived example, but I wanted to draw from the previous few example sections.

**Review by stargaser — APPROVED** [review]
Thank you for addressing the minor points. Looks good to me!

**Pull request merged**